### ISSUE TITLE
Chrome extension crashes Edge (chromium) when trying to fill in sign up

### ISSUE THREAD

**Issue by surgicalcoder** [hoarder-app/hoarder#763]
### Describe the Bug

Installed the latest version of the chrome extension  (1.2.3) onto Edge version Version 131.0.2903.112 (Official build) (64-bit) and when trying to fill in the URL for my instance of hoarder, it crashes the entire browser, forcing a restart

### Steps to Reproduce

1. Install Edge
2. Install Hoarder plugin
3. Click on the plugin to start sign up
4. Click on the text box to enter url of instance, try to type in

### Expected Behaviour

Not to crash the browser

### Device Details

Edge 131.0.2903.112 on Windows 11 23H2 (22631.4602)

### Exact Hoarder Version

v0.20.0

**Comment by MohamedBassem** [comment]
Wow. Crashing the entire browser smells like an edge problem rather than a hoarder problem. I'd be super surprised if extensions are allowed to crash entire browsers that way.

**Comment by surgicalcoder** [comment]
Yeah I was really surprised too, I've tried disabling all extensions I have on here, no luck. 

Interestingly though it only happens on Edge release version, not Dev or Canary.

It's not being reported as an app crash in terms of being picked up by WER. Maybe I was too hasty in opening this bug...

**Comment by solemgar** [comment]
Experiencing the same behavior. This time on MacOS so no OS dependent.

**Comment by kamtschatka** [comment]
I can not reproduce this issue and I honestly don't see how hoarder would be causing this.
I would rather assume there is an issue in Edge/Chromium and it should be reported in their bugtracker instead.

**Comment by solemgar** [comment]
@kamtschatka thanks for the quick reply. Are you trying on a edge browser release ? 
Is it possible to set the server address other way so we can test if the extension still works?

**Comment by solemgar** [comment]
@kamtschatka to close the loop here, the extension is working normally now, so definitely was some kind of problem on Edge browser, spot on :)
